reports/C0140/get_C0140_ao_data.rb

```diff
@@ -0,0 +1,413 @@
+# frozen_string_literal: true
+
+require 'archivesspace/client'
+require 'json'
+require 'nokogiri'
+require_relative '../../helper_methods'
+
+def remove_tags(text)
```

Comment:
This method seems opaque to me.  Could you add a comment showing a typical input and output?

Reply:
Ah, I remember! So for reasons to do with pulfalight display, some of the ASpace note fields contain EAD markup (like `<p>` tags and `<chronlist>` for example). The EAD markup breaks the marcXML export, so this method removes them. I'll push a comment.